app/workers/broadcasts/send_welcome_notifications_worker.rb

```diff
@@ -10,12 +10,14 @@ def perform
       # and will ultimately be superseded by 7.days.ago when it's larger than feature_live_date.
       return unless Settings::General.welcome_notifications_live_at
 
-      notifications_live_at = Settings::General.welcome_notifications_live_at
-      week_ago = 7.days.ago
-      latest_date = notifications_live_at > week_ago ? notifications_live_at : week_ago
-      User.select(:id).where("created_at > ?", latest_date).find_each do |user|
+      User.select(:id).where("created_at > ?", created_after).find_each do |user|
         Broadcasts::WelcomeNotification::Generator.call(user.id)
       end
     end
+
+    def created_after
+      [Settings::General.welcome_notifications_live_at, 7.days.ago].max
```

Comment:
this 7.days should _probably_ be 8 days if we ever want to send the download app notification (which filters out users newer than 7 days old).